src/base/reconstruction.cc

```diff
@@ -767,9 +767,9 @@ void Reconstruction::WriteCameras(const std::string& path) const {
   std::ofstream file(path, std::ios::trunc);
   CHECK(file.is_open()) << path;
 
-  file << "# Camera list with one line of data per camera:" << std::endl;
-  file << "#   CAMERA_ID, MODEL, WIDTH, HEIGHT, PARAMS[]" << std::endl;
-  file << "# Number of cameras: " << cameras_.size() << std::endl;
+  file << "# Camera list with one line of data per camera:\n";
```

Comment:
@ahojnnes I swapped most of ```std::endl```s for ```\n``` since ```endl``` does a flush, which can be an expensive system call. Instead I do a flush at the end of the method.